tests/integration/project/project_tests_test.go

```diff
@@ -0,0 +1,19 @@
+package project
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo project command tests", func() {
+
+	var commonVar helper.CommonVar
```

Comment:
Failure is causing due to `commonVar` is not accessible across the project package. 
I will fix it soon.